toolchain/sem_ir/facet_type_info.h

```diff
@@ -77,7 +77,7 @@ struct FacetTypeInfo : Printable<FacetTypeInfo> {
   // canonicalized, so that it can be further modified by the caller if desired.
   static auto Combined(const FacetTypeInfo& lhs, const FacetTypeInfo& rhs)
       -> FacetTypeInfo {
-    FacetTypeInfo info;
+    auto info = FacetTypeInfo{.other_requirements = false};
```

Comment:
```suggestion
  FacetTypeInfo info = {.other_requirements = false};
```

See initialization at https://github.com/carbon-language/carbon-lang/blob/trunk/docs/project/cpp_style_guide.md

Reply:
Done